package.json

```diff
@@ -42,6 +42,10 @@
     "trezor-connect": "^8.1.16",
     "typescript": "~4.0.5"
   },
+  "resolutions": {
+    "**/@typescript-eslint/eslint-plugin": "^4.1.1",
+    "**/@typescript-eslint/parser": "^4.1.1"
+  },
```

Comment:
I ran into a very weird bug with `react-scripts`. It was falsely throwing an eslint error in a module exporting an array. What I've done here is kind of a band aid fix: it upgrades the versions of typescript-eslint that `react-scripts` uses. Another option is to update `react-scripts` itself BUT that causes a whole host of other eslint issues to crop up which were out of scope of this issue. This was the least invasive method I was able to find.

Further reading here:
https://stackoverflow.com/a/63919395
https://github.com/facebook/create-react-app/issues/9515

Reply:
Yeah Haven't seen `resolutions` before but that seems like a useful tool for the toolbox!